### DISCUSSION TITLE
Contatto pulito a batteria

### DISCUSSION THREAD

**Discussion by AndreaFeira** [indomus/forum#555] [GENERALE - Discussioni generiche]
Salve a tutti, avrei bisogno di capire se è possibile realizzare, o esiste, un contatto pulito alimentabile da una batteria a 12 V.
Dovrei automatizzare il cancello automatico collegando un attuatore direttamente al radiocomando però non ho la possibilità di alimentarlo dalla rete elettrica. Esiste una soluzione per questa casistica?

Grazie in anticipo

**Comment by 50494554524F** [comment]
non lo so, ma ci sono questi aggeggetti... ne trovi diversi simili

https://it.aliexpress.com/item/1005002239223723.html?ug_edm_item_id=1005002239223723&creative_img_ind=2&creative_img_tpl=136626&tracelog=rowan&rowan_id1=spaml_product_2_picture_square_2_1_it_IT_2021-04-19&rowan_msg_id=_pieCOWS_209_$78ef604d69a240348cdf3cd40ee1e567&ck=in_edm_other

non li ho ancora provati, ma ho in programma di comprarli...
